src/Analyzers/MSTest.Analyzers.CodeFixes/Helpers/Utils.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Analyzer.Utilities;
+
+using Microsoft.CodeAnalysis;
+
+namespace MSTest.Analyzers.Helpers;
+
+internal class Utils
```

Comment:
Let's rename it to `CompilationExtensions`.

Reply:
we have the same naming in the analyzer project and cause conflict

Reply:
In that case the helper should be drop and you should use `semanticModel.Compilation.CanDiscoverInternals`